src/docker.py

```diff
@@ -65,8 +65,9 @@ def __extract_from_docker(self, docker_image: str, docker_binary_path: str, *arg
         sp.run(['docker', 'create', '--name', 'tmp', docker_image_and_tag], check=False)
         utils.stop_polkadot()
         sp.run(['docker', 'cp', f'tmp:{docker_binary_path}', utils.BINARY_PATH], check=True)
-        for command in args:
-            sp.run(command.split(' '), check=True)
+        if docker_specs_path:
+            sp.run(['docker', 'cp', f'tmp:{docker_specs_path}', utils.HOME_PATH], check=True)
+            sp.run(['chown', '-R', 'polkadot:polkadot', Path(utils.HOME_PATH, os.path.basename(docker_specs_path))], check=True)
```

Comment:
Why not use `Path` for `docker_specs_path`?

Imo you should try not to mix the two (`pathlib` and `os.path`), better to stick with one path lib.

`Path(utils.HOME_PATH, os.path.basename(docker_specs_path))` == `utils.HOME_PATH / Path(docker_specs_path).name`